zipkin-example/src/main/java/com/example/helloworld/HelloWorldApplication.java

```diff
@@ -19,7 +21,7 @@ public static void main(String[] args) throws Exception {
 
     @Override
     public String getName() {
-        return "hello-world";
+        return "dropwizard-zipkin";
```

Comment:
I think with the name `dropwizard` it is easier to identify the app. If you want I can change it.